### ISSUE TITLE
Staging Review finding: Summary screen Edit buttons have generic duplicate text

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#124610]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Intake Optimization (Aquia)
**Product:** VA Forms 21P-530a
**Feature:** MVP Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User experience is not consistent with the VA.gov design principles.](https://design.va.gov/about/experience-standards/#users-experience-is-consistent-with-the-vagov-design-principles)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

All Edit buttons on the Summary screen do not provide any additional context through additional labelling. Additional context is necessary, so that screen reader users  can clearly identify what the Edit button does without having to explore the surrounding content. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="387" height="508" alt="Edit buttons" src="https://github.com/user-attachments/assets/3bf178a6-09e5-4575-b136-1fe47b9c9ccd" />

</details>

## Recommended action

Additional button context is provided via the component typically. 
`<va-button uswds="true" aria-label="Edit Your organization's information" secondary="" text="Edit" class="hydrated">...`

However, either the implementation of the component is incorrect, or something is rendering incorrectly, as the additional context is not being announced to screen readers (tested across multuple screen readers)

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG22/#name-role-value)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by dluetger** [comment]
This appears to be corrected as of our most recent release, and likely before. I verified the button labels and with VoiceOver in Chrome.
